### PR TITLE
Add more instructions of publicAPI analyzer with nullability

### PR DESCRIPTION
**Pull request by jcouv**
Tagging @russkie @mavasani

### CODE REVIEW

File: src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md [hunk 1]
```diff
@@ -9,5 +9,5 @@ Rule ID | Title | Category | Enabled | Severity | CodeFix | Description |
...
-[RS0037](https://github.com/dotnet/roslyn-analyzers/blob/master/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) | Annotate nullability of types and members in the declared API | ApiDesign | True | Warning | True | All public types and members should be declared with nullability annotations in PublicAPI.txt. This draws attention to API nullability changes in the code reviews and source control history, and helps prevent breaking changes. |
+[RS0037](https://github.com/dotnet/roslyn-analyzers/blob/master/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) | Annotate nullability of types and members in the declared API | ApiDesign | True | Warning | True | PublicAPI.txt is missing '#nullable enable', so the nullability annotations of API isn't recorded. It is recommended to enable this tracking. |
```
Comment (mavasani) [inline src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md:12]: Did you update this file manually? This file is auto-updated when you run “Pack” command for the PublicApi analyzer Package.csproj, either from VS solution explorer or using “/t:pack” from command line build. The string used here is the one that is used in the desciptor’s Description resource string.

Comment (jcouv) [reply src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md:12]: Ah, okay. I'll revert then. I'm preparing a separate PR to fix some resx strings too. Will run Pack on that

Comment (jcouv) [reply src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md]: Reverted

**Comment by mavasani** [comment]
@jcouv ready to merge

**Pull request merged**